service/src/main.rs

```diff
@@ -751,34 +786,24 @@ fn ensure_account_has_funds(api: &mut Api<sr25519::Pair, WsRpcClient>, accountid
 	let nonce = api.get_nonce_of(&alice_acc).unwrap();
 	info!("    Alice's Account Nonce is {}", nonce);
 
-	// check account balance
-	let free = api.get_free_balance(accountid).unwrap();
-	info!("TEE's free balance = {:?}", free);
-
-	let existential_deposit = api.get_existential_deposit().unwrap();
-	info!("Existential deposit is = {:?}", existential_deposit);
-	let funding_amount = existential_deposit * MIN_FUND_INCREASE_FACTOR - free;
-	info!("Funding amount is = {:?}", funding_amount);
 	if funding_amount > alice_free.unwrap() {
 		error!(
```

Comment:
I think you could return here directly (respective return an error or panic, depending on where you're placing the error handling). Trying to transfer some funds after this check is a waste of time.